js/analyzer.js

```diff
@@ -1,8 +1,9 @@
 /**
- * Text Analyzer, accepts args for config and
+ * Text Analyzer, accepts args for config and calls init for initialisation
```

Comment:
I would use `initialization` because I thought that was the US-English way to that. Correct me if I'm wrong